packages/devtools_server/lib/src/server.dart

```diff
@@ -24,6 +26,8 @@ const launchDevToolsService = 'launchDevTools';
 
 const errorLaunchingBrowserCode = 500;
 
+final ClientManager clients = new ClientManager();
```

Comment:
nit: run dartfmt --fix on this file to eliminate uses of new here and elsewhere.

Reply:
Should/can we enable the `unnecessary_new` lint so we can see these in the editor?

Reply:
make it so!

Reply:
https://github.com/flutter/devtools/pull/998

FWIW, when I ran `dartfmt --fix` there was no output (and no changed files) after many minutes, so I terminated it and did it manually... is it expected to be really slow, or is that a bug?

Reply:
sorry I left out a few of the arguments you need to pass to `dartfmt fix`.

`dartfmt --fix  -w directory/to/run/on/`